db/typings.d.ts

```diff
@@ -1,6 +1,11 @@
 import type { Pool } from 'pg'
 
-export interface pgPools {
+export interface PgPools {
   stats: Pool;
   evaluate: Pool;
 }
+
+export interface EndablePgPools extends PgPools {
+  end(): Promise<unknown>
```

Comment:
Our implementation returns `Promise<[void, void]>`. Is it worth changing the implementation so that this line can describe the return value as `void`?

```suggestion
  end(): Promise<void>
```

Reply:
sgtm! should be a simple change

Reply:
Done in [4741c76](https://github.com/filecoin-station/spark-stats/pull/144/commits/4741c7660a1c230d45d1e5602e862c141511d427)